src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdAuthenticatorFactoryTest.java

```diff
@@ -27,23 +23,25 @@
     @Mock
     private AuthTokenGenerator tokenGenerator;
     @Mock
-    private IdamClient idamClient;
-    @Mock
-    private JurisdictionToUserMapping users;
+    private IdamCachedClient idamClient;
 
     private CcdAuthenticatorFactory service;
 
     @BeforeEach
     void before() {
-        service = new CcdAuthenticatorFactory(tokenGenerator, idamClient, users);
+        service = new CcdAuthenticatorFactory(tokenGenerator, idamClient);
     }
 
     @Test
     void should_sucessfully_return_authInfo() {
-        given(users.getUser(eq(JURSIDICTION))).willReturn(USER_CREDS);
+        CachedIdamCredential cachedIdamCredential = new CachedIdamCredential(
+            USER_TOKEN,
+            USER_DETAILS,
+            28800L
```

Comment:
Would be nice to define a constant explaining this value.

Reply:
it is just used 1 test.